src/components/Data/Documents/Page.vue

```diff
@@ -324,8 +324,16 @@ export default {
     listMappingGeopoints(mapping, path = []) {
       let attributes = []
 
-      for (const [attributeName, { type }] of Object.entries(mapping)) {
-        if (type === 'geo_point') {
+      for (const [attributeName, { type, properties }] of Object.entries(mapping)) {
+        if (properties) {
+          if (properties.lat && properties.lon) {
+            attributes = attributes.concat(path.concat(attributeName).join('.'))
+          }
+
+          attributes = attributes.concat(
```

Comment:
Why create a new array from scratch every time we want to add a value to the local `attributes` array, instead of just pushing the value in the existing array?

`attributes.push(this.listMappingGeopoints(properties, path.concat(attributeName)));`

Reply:
Because `listMappingGeopoints` return an array and I will end up with an array of array

Reply:
You can always spread it: `attributes.push(...this.listMappingGeopoints(properties, path.concat(attributeName)));`